README.md

```diff
@@ -116,6 +116,47 @@ PAF type definitions are organized into classes. Classes are currently not used
 
 When type name references are required in the PAF file, two forms are applicable. Either a class name can be given, which will include all types of that class or a ClassName.TypeName form can be used to reference a single type only.
 
+## Analysis Modes
+An analysis is one or a series of operations which uses QTM to generate output data. It can use QTM measurements in the data tree to do calculations and export files. Exactly what the analysis does and which files it uses is defined in the PAF YAML file. See [Overview of analyses](#overview-of-analyses) for more information about analyses.
+
+Two modes of execution of analyses are supported, these are defined by their relation to their measurement files. 
+
+1. ``Session level analysis``
+    In this mode measurement files are placed directly under the folder the analysis.
+
+2. ``Parent sessions level analysis``
+    In this mode the analsysis is executed on a folder that does not directly contain measurement files. Instead it contains Subsession folders, which in turn contain the measurement files.
+    This is useful when you need to create bulk analyses on a large amount of files.
+
+These modes also affects if certain analyses and analysis features are a available. This is described per- analysis.
```

Comment:
Consider changing the word "affects" to "determine". Alternatively, if you wish to keep the word "affect", remove the 's' at the end to be grammatically correct.